ctrd/image.go

```diff
@@ -248,6 +249,15 @@ outer:
 	}
 }
 
+func truncateID(id string) string {
+	const shortLen = 12
```

Comment:
Actually we did not recommend add a const variable in func. 
Since it is only used here, I suggest that we can just define it with `var shortLen = 12`.
WDYT? @0x04C2